app/main/controllers/attachments-ctrl.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+angular.module('myApp')
+.controller('AttachmentsCtrl', function ($scope, $state, $rootScope, DataService) {
+
+    $scope.nextButton = function() {
+        $state.go('review-submission');
```

Comment:
Update this to add the appId; it should be `$state.go('review-submission', {appId: $rootScope.application._id);` IIRC. You can test this by trying the Next button and telling whether it sends you to the first step of the app or actually to the review page.